docs/scenarios/parametric.md

```diff
@@ -68,15 +68,10 @@ The following dependencies are required to run the tests locally:
 - Docker
 - Python 3.12
 
-then, run the following command, which will create a Python virtual environment and install the Python dependencies from the root directory:
-
-```sh
-./build.sh -i runner
```

Comment:
Why removing this?

Reply:
The run command automatically builds so I thought it was not needed. Let me know if you think I should add it back!

Reply:
the run command only build the tested container. `./build.sh -i runner` builds the python virtual env used in ./run.sh`, so it's needed. Though, you need it only once, until we don't update system-tests requirements.

Reply:
I'll add it back! Thank you